src/plugins/kernel_install/utils.sh

```diff
@@ -630,6 +630,12 @@ function install_kernel()
 
   install_modules "$target" "$flag"
 
+  # Copy dtbs if exist
+  if ls "${KW_DEPLOY_TMP_FILE}"/kw_pkg/*.dtb > /dev/null 2>&1; then
```

Comment:
Avoid `ls` since this command does not have a standard behavior across distros. I recommend you to use `find`. Also, avoid bash commands inside the if condition.

Reply:
I just updated the commit message and the code to use `find`. I need more time to examine and create a test.